### ISSUE TITLE
rio secret add --docker not working

### ISSUE THREAD

**Issue by lucidprogrammer** [rancher/rio#728]
**Describe the bug**
Cannot add docker secret
**To Reproduce**
1. 
rio secret add --docker

`FATA[0011] Secret "dockerconfig" is invalid: [data[%s][username]: Required value, data[%s][password]: Required value]`

**Expected behavior**

**Kubernetes version & type (GKE, on-prem)**: `kubectl version`
```
lient Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.7-eks-1861c5", GitCommit:"1861c597586f84f1498a9f2151c78d8a6bf47814", GitTreeState:"clean", BuildDate:"2019-09-24T22:12:08Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.6-eks-5047ed", GitCommit:"5047edce664593832e9b889e447ac75ab104f527", GitTreeState:"clean", BuildDate:"2019-08-21T22:32:40Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
```
Type:
**Rio version**: `rio info`
```
Rio Version: v0.6.0-alpha.4 (969baf29)
Rio CLI Version: v0.6.0-alpha.4 (969baf29)

```
**Additional context**
`rio systemlogs`
output:
```

```

**Comment by StrongMonkey** [comment]
Fixed in https://github.com/rancher/rio/pulls

**Comment by StrongMonkey** [comment]
available to test in v0.6.0-alpha5.

**Comment by rancher-max** [comment]
I was able to recreate this on alpha4 and verified it is fixed in alpha5.
